src/nvim/eval/funcs.c

```diff
@@ -2157,6 +2157,43 @@ static void f_expandcmd(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   rettv->vval.v_string = cmdstr;
 }
 
+/*
+ * "flatten(list[, {maxdepth}])" function
+ */
```

Comment:
There is an error in `clint` as shown in the log below. Please amend this to a comment in `//` format.

https://travis-ci.org/github/neovim/neovim/jobs/703247240#L2373

Reply:
Sure, I just followed the local convention. I'll change that.